lib/generators/active_fedora/config/solr/templates/solr/config/solrconfig.xml

```diff
@@ -119,9 +119,6 @@
 
        <str name="facet">true</str>
        <str name="facet.mincount">1</str>
-       <str name="facet.limit">10</str>
-       <str name="facet.field">active_fedora_model_ssi</str>
```

Comment:
Did you mean to also remove these `facet.field` entries (not that I'm defending them..)?